### PR TITLE
various insight fixes

### PR DESCRIPTION
**Pull request by paulandrieux**
## Type
improvements

## Purpose
Some sensiolabs insight fixes

## BC Break
NO

### CODE REVIEW

**Comment by paulandrieux** [comment]
image

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [![Version](https://img.shields.io/packagist/v/Victoire/Victoire.svg)](https://packagist.org/packages/Victoire/Victoire)
 [![CircleCI](https://circleci.com/gh/Victoire/victoire.svg?style=shield)](https://circleci.com/gh/Victoire/victoire)
 [![Scrutinizer Quality Score](https://img.shields.io/scrutinizer/g/Victoire/Victoire.svg)](https://scrutinizer-ci.com/g/Victoire/victoire/)
-[![SensioLabsInsight](https://insight.sensiolabs.com/projects/d5307bf2-eac4-43db-bd49-dd9e85e360a5/mini.png)](https://insight.sensiolabs.com/projects/d5307bf2-eac4-43db-bd49-dd9e85e360a5)
+[![SensioLabsInsight](https://insight.sensiolabs.com/projects/92a98a39-8fa0-4e3c-b90d-e56d4c8f89b3/mini.png)](https://insight.sensiolabs.com/projects/92a98a39-8fa0-4e3c-b90d-e56d4c8f89b3)
```
Comment (talbotseb) [inline README.md]: Arte we sure that it is the good project analyzed ?

Comment (paulandrieux) [inline README.md]: Yes why ? Previously it was @lenybernard 's insight but I plugged mine to track the current 2.1 branch

Comment (paulandrieux) [inline README.md]: Yes why ? Previously it was @lenybernard 's insight but I plugged mine to track the current 2.1 branch

Comment (paulandrieux) [inline README.md]: Yes why ? Previously it was @lenybernard 's insight but I plugged mine to track the current 2.1 branch

**Pull request merged**